Examples/Io/Json/include/ActsExamples/Io/Json/JsonSurfacesReader.hpp

```diff
@@ -36,4 +36,12 @@ struct Options {
 Acts::GeometryHierarchyMap<std::shared_ptr<Acts::Surface>> read(
     const Options& options);
 
+/// @brief Read the sflat surfaces urfaces from the input file
+///
+/// @param inputFile is the input file to read from
+///
+/// @return  a vector of surfaces
+std::vector<std::shared_ptr<Acts::Surface>> readSurfaces(
```

Comment:
I think the function names could be a bit confusing:
`read()` - "Read the surfaces"
`readSurfaces()` - "Read the flat surfaces"

Reply:
Good point, as I saw that the second method is not referenced either by run script in the repository, I can rename them to `readSurfaceHierarchyMap` and `readSurfaceContainer`, however, then I will probably introduce a `acts.examples.json` module.

Reply:
done, submodule not yet included, this needs a general cleanup of the python modules.